docs/build-ansible.rst

```diff
@@ -224,6 +224,11 @@ schedule too).
 Announcing Ansible
 ~~~~~~~~~~~~~~~~~~
 
+.. note::
```

Comment:
This is fine for now, but I think we should finish up the a-b-d docs and remove this page entirely sooner rather than later. I'll open an issue.

Reply:
Ref: #614

Reply:
#616 deletes these docs, I'll rebase once that PR is merged.